compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3486,6 +3488,37 @@ TR::Register *OMR::Power::TreeEvaluator::vmulDoubleHelper(TR::Node *node, TR::Co
    return TR::TreeEvaluator::inlineVectorBinaryOp(node, cg, TR::InstOpCode::xvmuldp);
    }
 
+TR::Register *OMR::Power::TreeEvaluator::vmulInt16Helper(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
Please amend this commit to define `vmulInt16Helper` before `vmulInt32Helper` rather than reordering in the next commit. Similarly for the declaration in the header and for the `VectorInt16` case in `vmulEvaluator` above